src/main/java/io/jenkins/plugins/orka/RunOnceCloudRetentionStrategy.java

```diff
@@ -9,24 +9,29 @@
 import hudson.slaves.AbstractCloudSlave;
 import hudson.slaves.CloudRetentionStrategy;
 import hudson.slaves.RetentionStrategy;
+import hudson.util.FormValidation;
+import io.jenkins.plugins.orka.helpers.Utils;
 
 import java.io.IOException;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
 
 public class RunOnceCloudRetentionStrategy extends CloudRetentionStrategy implements ExecutorListener {
     private static final Logger LOGGER = Logger.getLogger(RunOnceCloudRetentionStrategy.class.getName());
 
-    private final int idleMinutes;
+    private int idleMinutes;
+    public static final int RECOMMENDED_MIN_IDLE = 1;
 
     @DataBoundConstructor
     public RunOnceCloudRetentionStrategy(int idleMinutes) {
-        super(idleMinutes);
-        this.idleMinutes = idleMinutes;
+        super(Utils.normalizeIdleTime(idleMinutes, RECOMMENDED_MIN_IDLE));
+        
+        this.idleMinutes = Utils.normalizeIdleTime(idleMinutes, RECOMMENDED_MIN_IDLE);
     }
-
+    
```

Comment:
This can be removed

Reply:
Will do. Gotta check my editor settings to see why it keeps inserting these.